Update README (#168)
tesseract-ocr/tesstrain: README.md

    PSM                Page segmentation mode. Default: 6
    RANDOM_SEED        Random seed for shuffling of the training data. Default: 0
    RATIO_TRAIN        Ratio of train / eval training data. Default: 0.90

>     TARGET_ERROR_RATE  Stop training if one of the error rates (word, char, etc.) gets below this value. Default: 0.01

Author: I picked this from the Tesseract 4 training guide (https://github.com/tesseract-ocr/tessdoc/blob/master/TrainingTesseract-4.00.md), where it says: `Stop training if the mean percent error rate gets below this value.` 

According to the output of `lstmtraining --help` it is the `Final error rate in percent`, without further explanations whether it is word or char or whatever . It's the official statement in this issue - maybe I should update the text according to this? 

From my limited training experiences, I guess of course your're right, because I've never seen a situation where character error rate was higher than word error rate.

This week I've seen the following training output:

```
At iteration 36115/700000/700000, Mean rms=0.062%, delta=0.004%, char train=0.01%, word train=0.053%, skip ratio=0%,  New best char error = 0.01 wrote best model:/home/gitlab-runner/builds/WhA4zmkC/0/ulb/ulb-dd-ocr-training/tesstrain/data/frk_ulbzd1/checkpoints/frk_ulbzd10.01_36115.checkpoint wrote checkpoint.
Finished! Error rate = 0.01
```
In this case the target error rate condition was met by char error. 
Therefore my guess: It's the first error rate that touches this limit.

>     TARGET_ERROR_RATE  Stop training if one of the error rates (word, char, etc.) gets below this value. Default: 0.01

Reviewer: From the code I'd expect it is only the CER: `best_error_rate_` is the CER (see [code](https://github.com/tesseract-ocr/tesseract/blob/master/src/training/lstmtrainer.cpp#L287) which assigned `error_rate`, used later with `best_error_rate_`), and it terminates the training (see [code](https://github.com/tesseract-ocr/tesseract/blob/master/src/training/lstmtraining.cpp#L216)).

>     TARGET_ERROR_RATE  Stop training if one of the error rates (word, char, etc.) gets below this value. Default: 0.01

Reviewer: ```suggestion
    TARGET_ERROR_RATE  Stop training if the character error rate (CER) in percent gets below this value. Default: 0.01
```

>     TARGET_ERROR_RATE  Stop training if one of the error rates (word, char, etc.) gets below this value. Default: 0.01

Reviewer: If you agree, I'd apply your pull request with my updated text.

>     TARGET_ERROR_RATE  Stop training if one of the error rates (word, char, etc.) gets below this value. Default: 0.01

Author: @stweil You're welcome to do so!

Since my knowledge on tesseract training, self-evaluation and lstm-networks is rather limited, I wonder why the comment near your link distinguishes between `best and locally worst char error_rate`. But anyway, it has to do with characters.
